packages/expo-camera/src/next/CameraView.tsx

```diff
@@ -139,9 +139,10 @@ export default class CameraView extends React.Component<CameraProps> {
   }
 
   /**
-   * Presents a modal view controller that uses the `DataScannerViewController` available on iOS 16+.
+   * Presents a modal view controller that uses the [`DataScannerViewController`](https://developer.apple.com/documentation/visionkit/scanning_data_with_the_camera) available on iOS 16+.
+   * @platform ios
    */
-  static async launchModernScanner(options?: ModernScanningOptions): Promise<void> {
+  static async launchScanner(options?: ModernScanningOptions): Promise<void> {
```

Comment:
```suggestion
  static async launchScanner(options?: ScanningOptions): Promise<void> {
```

Can we also rename the parameter type accordingly to method chnage?